daffodil-propgen/src/main/scala/org/apache/daffodil/propGen/TunableGenerator.scala

```diff
@@ -292,7 +292,7 @@ class EnumListTunable(
       "Nil"
     } else {
       val defaultSeq =
-        trimmedDefault.split("\\s+").map(d => s"${listType}.${s"${d.head.toUpper}${d.tail}"}")
+        trimmedDefault.split("\\s+").map(d => s"${listType}.${d.head.toUpper +: d.tail}")
```

Comment:
As above, this one is not just concatenating two strings. 
So in this case a single string interpolation is the right thing.
```
s"${listType}.${d.head.toUpper}${d.tail}"
```